annotationPipeline/pom.xml

```diff
@@ -4,12 +4,12 @@
   <name>cBioPortal Annotation Pipeline</name>
   <description>Annotation Pipeline</description>
   <artifactId>annotationPipeline</artifactId>
-  <version>0.1.0</version>
+  <version>1.0.0</version>
```

Comment:
I guess we have not been versioning the code yet, so this is good. But why a major revision increase? I guess it doesn't matter, but there is no fundamental change to the code, so we could use 0.2.0